File: libraries/onyxsdk-base/src/main/java/com/onyx/android/sdk/utils/MatrixUtils.java
package com.onyx.android.sdk.utils;

import android.graphics.Matrix;

/**
 * <pre>
 *     author : lxw
 *     time   : 2019/8/8 18:20
 *     desc   :
 * </pre>
 */
public class MatrixUtils {

    public static boolean isEmptyMatrix(Matrix matrix) {
        return matrix == null || matrix.equals(new Matrix());
    }

    public static float[] getMatrixValue(Matrix matrix) {
        float[] values = new float[9];
        matrix.getValues(values);
        return values;
    }

}

File: app/simple/Gallery/src/main/kotlin/com/onyx/gallery/utils/MatrixUtils.kt
package com.onyx.gallery.utils

import android.graphics.Matrix
import android.graphics.PointF

/**
 * Created by Leung 2020/9/10 15:18
 **/
object MatrixUtils {

    fun mapPointF(pointF: PointF, matrix: Matrix): PointF {
        val floatArray = floatArrayOf(pointF.x, pointF.y)
        matrix.mapPoints(floatArray)
        return PointF(floatArray[0], floatArray[1])
    }

}